frontend/app/src/components/accounts/AccountBalanceTable.vue

```diff
@@ -24,6 +24,42 @@
           @input="setSelected($event)"
         />
       </template>
+      <template #header.numOfDetectedTokens="{ header }">
+        <span>
+          <span>
+            {{ header.text }}
+          </span>
+          <span>
+            <v-tooltip top>
+              <template #activator="{ on }">
+                <v-btn
+                  text
+                  icon
+                  :disabled="detectingTokens().value || loading"
```

Comment:
Since we call this without any params both here and below wouldn't it make more sense to assign it to a const in the script so that we have free de-referencing?